README.md

```diff
@@ -10,7 +12,9 @@ $ QUANTUM_CUSTOMER=... ./ansible-openstack-inventory
 
 ## Todo
 
-- [ ] return correct JSON format of hosts
-- [ ] how to add additional groups
-- [ ] implement `--list`
+- [x] return correct JSON format of hosts
+- [x] how to add additional groups
+- [x] implement `--list`
+- [x] implement `--host node`
+- [ ] better error handling (instead of `os.Exit`)
```

Comment:
If this works reliably, I'll add a log library and then refactor this a bit. That's why I left it in there.